### DISCUSSION TITLE
Q: 无法使用vercel部署 || Q: Cannot be deployed using vercel

### DISCUSSION THREAD

**Discussion by Radisnt01** [walinejs/waline#2747] [Q&A] [answered]
### Describe the bug | 问题描述

通过[文档教程](https://waline.js.org/next/guide/deploy/vercel.html)部署时出现错误，我在vercel论坛上寻求过帮助但是没有收到任何回应
```
An unexpected error occurred. Our team has already been notified and are working to resolve the issue, please try again shortly.
```

![屏幕截图_5-10-2024_115347_vercel com](https://github.com/user-attachments/assets/86c4a0c9-ee55-4187-87e1-e6bdea68c9fe)
![image](https://github.com/user-attachments/assets/16833ded-5971-4afd-a23b-4bfd4939b7b5)

### Website URL | 问题网站

https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fwalinejs%2Fwaline%2Ftree%2Fmain%2Fexample

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

An error occurred when deploying through [Documentation Tutorial](https://waline.js.org/next/guide/deploy/vercel.html). I asked for help on the vercel forum but did not receive any response.
```
An unexpected error occurred. Our team has already been notified and are working to resolve the issue, please try again shortly.
```

![Screenshot_5-10-2024_115347_vercel com](https://github.com/user-attachments/assets/86c4a0c9-ee55-4187-87e1-e6bdea68c9fe)
![image](https://github.com/user-attachments/assets/16833ded-5971-4afd-a23b-4bfd4939b7b5)

### Website URL | Problem website

https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fwalinejs%2Fwaline%2Ftree%2Fmain%2Fexample

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is the data stored?

LeanCloud(https://leancloud.app)

**Comment by Ryliey** [comment, marked as answer]
取消勾选"Keep my email addresses private"
https://github.com/settings/emails
![Github_Settings](https://github.com/user-attachments/assets/bba206b1-2347-4e63-b8ee-af61d39c209d)
